mockup/js/bundles/structure.js

```diff
@@ -22,7 +22,7 @@ define([
         // uh oh, no content id, let's go up a few levels and use that as parent
         $container = $structure.parent().parent();
       }
-      var $modal = $container.patternModal({
+      var $modal = $container.patPloneModal({
```

Comment:
Currently the registry supports both patternName and patName.

In Patternslib, patternName is legacy and patName is the newer way. I saw that Mockup uses patternName when I did the merging work at the Alpine sprint.

I personally prefer patName just because it's shorter and the patterns are named like that as well (e.g. "pat-name").

Not a big deal though, can also be renamed if required.